### ISSUE TITLE
Read the docs bug?

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#630]
Within PCT 1.2.0, going to the documentation reveals the following:

![image](https://cloud.githubusercontent.com/assets/7317450/21991842/08d4ad9c-dbc9-11e6-8f9a-7b0ba1606de3.png)

I think the problem might be in the version number; it should be pointing to:
http://corpustools.readthedocs.io/en/v1.2/functional_load.html

which is v1.2, not v1.2.0 -- how do we change this?

**Comment by kchall** [comment]
Hmm, possibly related:

I notice that the "latest" version of the Read the Docs isn't actually up to date. Specifically, http://corpustools.readthedocs.io/en/latest/functional_load.html (and even the 1.2 equivalent version, i.e., http://corpustools.readthedocs.io/en/v1.2/functional_load.html) doesn't have the updated information about the option for normalizing the delta-H algorithm. 

I had updated the version on the develop branch (https://github.com/PhonologicalCorpusTools/CorpusTools/blob/develop/docs/source/functional_load.rst) but it doesn't seem like it got pulled in when the Read the Docs was updated? 

(There may be other discrepancies, too -- this is just the one I noticed!)
